docs/tutorials/intro.md

```diff
@@ -132,17 +130,17 @@ use icu::datetime::{DateTimeFormatter, options::length};
 const LOCALE: Locale = locale!("ja"); // let's try some other language
 
 fn main() {
-    let provider = icu_testdata::any();
+    let provider = icu_testdata::unstable();
```

Comment:
I think this is better with `AnyProvider`. `unstable` sounds scary, and explaining what it means, how it's more efficient, how it supports data slicing, is beyond the scope of the intro. Any is "nice".

Reply:
I wish we could agree on a better term than `unstable` because it really shouldn't be that scary. But ok.

Reply:
Maybe `versioned` would have been a better name?